packages/calcite-components/src/components/action-menu/action-menu.tsx

```diff
@@ -334,7 +336,10 @@ export class ActionMenu extends LitElement implements LoadableComponent {
 
   private setDefaultMenuButtonEl(el: Action["el"]): void {
     this.defaultMenuButtonEl = el;
-    this.connectMenuButtonEl();
+
+    if (el) {
```

Comment:
Seems like a lot of components might need this kind of if statement since this is similar to some other fixes 👍

Reply:
Agreed. I created https://github.com/Esri/calcite-design-system/issues/11093 to take a closer look, but it might be easier for the time being to return early when the reference element is null in all cases (preserves existing component logic based on Stencil lifecycle). WDYT?

Reply:
Yeah I think that might be the easiest and most consistent solution.